api/Window.json

```diff
@@ -1560,10 +1560,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/crypto",
           "support": {
             "chrome": {
-              "version_added": "37"
+              "version_added": "1"
```

Comment:
This can't be right, `window.crypto` is newer than this. https://trac.webkit.org/changeset/78321/webkit is when it was added to WebKit, although there was something before that returning undefined with a "FIXME: Implement me" comment. I didn't check how that that was around for.

This also threw off results for Safari, so maybe we should do a custom test for `window.crypto` that ensures it exists *and* is an object at least?

Reply:
https://github.com/mdn/browser-compat-data/pull/6825#discussion_r501629768 the other place we talked about this.